projects/core/src/error-handling/effects-error-handler/effects-error-handler.service.ts

```diff
@@ -44,4 +44,17 @@ export class EffectsErrorHandlerService {
   filterActions(action: Action): action is ErrorAction {
     return 'error' in action;
   }
+
+  /**
+   * Determine if the error should be handled by the `ErrorHandler`.
+   *
+   * Be default, we avoid sending unpredictable errors to the browser's console, to prevent
+   * possibly exposing there potentially confidential user's data.
+   * This isn't an issue in SSR, where pages are rendered anonymously.
+   * Moreover, in SSR we want to capture all app's errors, so we can potentially send
+   * a HTTP error response (e.g. 500 error page) from SSR to a client.
+   */
+  protected shouldHandleError(): boolean {
```

Comment:
For better extensibility, let's pass `_error: unknown` as a param to this method. 
Thanks to this, customers may implement a custom logic, that let-through only some specific error objects.

note: this comment applies also to other `shouldHandleError` method in this PR